app/controllers/concerns/bbb_server.rb

```diff
@@ -45,8 +45,15 @@ def get_multiple_recordings(meeting_ids)
   # Returns a URL to join a user into a meeting.
   def join_path(room, name, options = {}, uid = nil)
     # Create the meeting, even if it's running
-    start_session(room, options)
-
+    retries = 0
+    begin
+      start_session(room, options)
+    rescue StandardError => e
+      retries += 1
+      logger.warn "Retrying #{retries} time(s)."
```

Comment:
Same here we need a clear context for the logs.

Reply:
Sure,